doc/en/api/1.0.0/gwcdiskquota.yaml

```diff
@@ -0,0 +1,50 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache DiskQuota
+  description: The REST API for Disk Quota management provides a RESTful interface through which clients can configure the disk usage limits and expiration policies for a GeoWebCache instance through simple HTTP calls.
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /diskquota:
+    get:
+      produces:
+        - application/xml
+        - application/json
+      operationId: diskQuotaGet
+      summary: Returns the gwc Quota configuration
+      description: Returns the GeoWebCache DiskQuota configurations which include disk usage limits and expritation policies for a gwc instance.
+      responses:
+        200:
+          description: OK
+          examples:
+            application/xml: |
+              <gwcQuotaConfiguration>
+                <enabled>true</enabled>
+                <diskBlockSize>2048</diskBlockSize>
+                <cacheCleanUpFrequency>5</cacheCleanUpFrequency>
+                <cacheCleanUpUnits>SECONDS</cacheCleanUpUnits>
+                <maxConcurrentCleanUps>5</maxConcurrentCleanUps>
+                <globalExpirationPolicyName>LRU</globalExpirationPolicyName>
+                <globalQuota>
+                  <value>100</value>
+                  <units>MiB</units>
+                </globalQuota>
+                <layerQuotas/>
+              </gwcQuotaConfiguration>
+    put:
+      consumes:
+        - application/xml
+        - application/json
+      operationId: diskQuotaPut
+      summary: Modify properties of a gwc instance disk quota configuration.
+      description:  The request body for PUT should contain desired properties to be modified.
+      responses:
+        200:
+          description: OK
+          examples:
+            application/json: |
+              {"gwcQuotaConfiguration":{"diskBlockSize":2048,"enabled":true,"globalQuota":{"value":"100","units":"MiB"}}
```

Comment:
Is this the content of the response, or the request? If the request, it belongs in the request body parameter, not in the response examples.
If the response, then you should add a produces section since the response has content.